### PR TITLE
add slave-label explanation to IS/IST -> pod template

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees @bmcelvee @dongboyan77

### CODE REVIEW

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -458,9 +458,17 @@ with the appropriate  annotation, it generates the corresponding Kubernetes
...
+The name and image references of the image stream or image stream tag are mapped
+to the name and image fields in the Kubernetes plug-in pod template.  The label field of
+the Kubernetes plug-in pod template can be controlled by setting an annotation on the
+image stream or image stream tag object with the key "slave-label".  Otherwise, the name is used
+as the label.
+
 When it finds a config map with the appropriate label, it assumes the any values
 in the key-value data payload of the config map contains XML consistent with the
-config format for Jenkins and the Kubernetes plug-in pod templates.
+config format for Jenkins and the Kubernetes plug-in pod templates.  A key differentiator then
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/then/to/

**Comment by bparees** [comment]
one nit and lgtm

File: using_images/other_images/jenkins.adoc (hunk 1 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: thx update pushed

**Comment by bmcelvee** [comment]
LGTM. I'm going to open a follow-up PR with minor edits.

**Comment by gabemontero** [comment]
thanks @bmcelvee

**Comment by bmcelvee** [comment]
Sure thing, @gabemontero!

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../using_images/other_images/jenkins.adoc#using-images-other-images-jenkins[Jenkins]
|Added `slave-label` explaination to the image stream or image stream tag in the pod template in the xref:../using_images/other_images/jenkins.adoc#using-the-jenkins-kubernetes-plug-in-to-run-jobs[Using the Jenkins Kubernetes Plug-in to Run Jobs] section.
%

**Pull request merged**